src/videojs-contrib-hls.js

```diff
@@ -104,10 +105,12 @@ export default class HlsHandler extends Component {
 
     this.on(this.tech_, 'play', this.play);
 
-    this.tech_.audioTracks().addEventListener('change', () => {
-      for (let i = 0; i < this.tech_.audioTracks().length; i++) {
-        if (this.tech_.audioTracks()[i].enabled) {
-          return this.masterPlaylistController_.useAudio(this.tech_.audioTracks()[i].label);
+    let audioTrackList = this.tech_.audioTracks();
+
+    audioTrackList.addEventListener('change', () => {
+      for (let i = 0; i < audioTrackList.length; i++) {
+        if (audioTrackList[i].enabled) {
+          return this.masterPlaylistController_.useAudio(audioTrackList[i].label);
```

Comment:
With the changes in `useAudio` this while handler can just be a call to useAudio without any arguments and it'll automatically select the enabled audio track.